### DISCUSSION TITLE
Read Table of contents links data or show separate outline.

### DISCUSSION THREAD

**Discussion by qaisershehzad** [wojtekmaj/react-pdf#992] [Q&A]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-PDF version
- [X] I have checked if this bug is not already reported
- [X] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
- [ ] If I have a problem with PDF rendering, I checked if my PDF renders properly in [PDF.js demo](https://mozilla.github.io/pdf.js/web/viewer.html)

### Description

Hello I need help with Table of Contents of a PDF file.

I have a PDF file with table of Contents which working fine normally. But I want to provide side menu from where user can navigate into PDF file. Left Outline like thing. Is there a way I can read Table of Contents of the PDF file so that I can generate a separate view for Table of contents. In PDF link navigation is fine.. 

As in below screenshot left side links from where user can navigate in PDF.
[sample.pdf](https://github.com/wojtekmaj/react-pdf/files/8493759/sample.pdf)
<img width="1157" alt="Screenshot 2022-04-15 at 9 08 31 AM" src="https://user-images.githubusercontent.com/11673425/163516970-74298b8a-6a8e-4822-8c91-26c8cd65f470.png">

### Steps to reproduce

You can try to load attached sample.pdf

### Expected behavior

Looking for table of contents links information

### Actual behavior

Not able to read links information.

### Environment

- **Browser (if applicable)**:
- **React-PDF version**:
- **React version**:
- **Webpack version (if applicable)**:
latest version
react 17

**Comment by ricardoc05** [comment]
Below I copy the piece of code that I use to display the table of contents in a side panel, but I still haven't figured out how to style it the way I want

` <Grid xl={6}>
                            <Box sx={{ my: 4 }}>
                                <Document file={file} onLoadSuccess={onDocumentLoadSuccess} options={options} >
                                    <Drawer
                                        variant="persistent"
                                        anchor="left"
                                        open={state}>
                                        <DrawerHeader>
                                            <Typography variant="body3" display="block" color='black' gutterBottom>
                                                INDICE
                                            </Typography>
                                            <IconButton onClick={TOCClose}>
                                                <ChevronLeftIcon />
                                            </IconButton>
                                        </DrawerHeader>
                                        <Divider />
                                                <Outline onItemClick={clickTOC} onLoadSuccess={(outline) => toc(outline)}   />
                                    </Drawer>
                                    <Page key={`page_${page}`} pageNumber={page} onRenderSuccess={onPageRenderSuccess} 
                                        customTextRenderer={textRenderer}
                                        renderTextLayer="true" />
                                </Document>
                                <Pagination page={page} count={numPages} color="primary" onChange={(e, page) => setPage(page)} />
                            </Box>
                        </Grid>
`
